### PR TITLE
Transcriptions ajax

### PR DESCRIPTION
**Pull request by chompomonim**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -78,6 +78,7 @@ Vagrant.configure(2) do |config|
       grant all privileges on *.* to 'manoseimas'@'localhost';
       flush privileges;" || true
     sudo apt-get install -y nodejs nodejs-legacy npm
+    sudo apt-get install -y coffeescript
```
Comment (frgtn) [inline Vagrantfile]: Deployment script also needs to install coffeescript compiler. See https://github.com/ManoSeimas/manoseimas.lt/blob/master/deployment/deploy.yml#L41

File: manoseimas/mps_v2/views.py [hunk 2]
```diff
@@ -108,6 +108,8 @@ def mp_fraction(request, fraction_slug):
     return render(request, 'fraction.jade', context)
 
 
+
```
Comment (frgtn) [inline manoseimas/mps_v2/views.py]: This is whitespace police says you have too many blank lines between functions (should be 2).

File: manoseimas/mps_v2/views.py [hunk 3]
```diff
@@ -246,3 +237,35 @@ def mp_discussion(request, statement_id):
 def mp_discussion_json(request, statement_id):
     context = _build_discussion_context(request, statement_id)
     return JsonResponse(context)
+
+
+def mp_statements(request, mp_slug, statement_page=None):
+    mp = ParliamentMember.objects.get(slug=mp_slug)
+
+    selected_session = request.GET.get('session')
+    # all_sessions = Stenogram.objects.distinct().values('session')
```
Comment (frgtn) [inline manoseimas/mps_v2/views.py]: Might as well remove this commented out code.

File: manoseimas/settings/base.py [hunk 4]
```diff
@@ -82,6 +82,7 @@
 )
 
 COMPRESS_PRECOMPILERS = (
+    ('text/coffeescript', 'coffee --compile --stdio'),
```
Comment (frgtn) [inline manoseimas/settings/base.py]: I might argue that all custom mimetypes should be prefixed with `x-` like (`type/x-coffeescript`) but people on the Internets already do it without the prefix.

**Comment by frgtn** [comment]
Only real issue is `coffeescript` not being installed when deploying.

File: Vagrantfile (hunk 1 above)
Comment (sirex) [inline Vagrantfile]: CoffeeScript should be already installed using `npm install manoseimas/widget/frontend/` from https://github.com/ManoSeimas/manoseimas.lt/blob/master/manoseimas/widget/frontend/package.json with pinned versions.

**Comment by frgtn** [comment]
Merging as I need these changes for another feature.

**Pull request merged**